src/events/guild/roles/roleUpdate.ts

```diff
@@ -15,11 +18,15 @@ export default class extends Event {
 	}
 
 	public async run(oldRole: Role, newRole: Role): Promise<void> {
-		const difference = this.client.bulbutils.diff(oldRole, newRole).filter(k => k !== "rawPosition")
-		if(difference.length === 0) return;
+		const config: GuildConfiguration = await databaseManager.getConfig(newRole.guild.id);
+		if (newRole.rawPosition > newRole.guild.me.roles.highest.rawPosition && newRole.id === config.muteRole) await databaseManager.setMuteRole(newRole.guild.id, null);
+		else if (newRole.rawPosition > newRole.guild.me.roles.highest.rawPosition && newRole.id === config.autorole) await databaseManager.setAutoRole(newRole.guild.id, null);
```

Comment:
Is there any reason we don't support having your mute role as your autorole? If not, this shouldn't be an `else`

Reply:
don't think we said anything about it, but I feel like there is no reason to set the muted role as the autorole in the server, can't find any usescase where that would be needed.